app/transcription.py

```
@@ -43,11 +48,20 @@ def __init__(
         needs_review=False,
         include_metadata=True,
         correct=False,
-        llm_provider="openai",
-        llm_correction_model="gpt-4o",
-        llm_summary_model="gpt-4o",
+        llm_provider=settings.LLM_PROVIDER,
+        llm_correction_model=settings.config.get("llm_correction_model", "gpt-4o"),
```

Ruff (0.15.12)

[warning] Do not perform function call `settings.config.get` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)